packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -733,9 +734,26 @@ export default {
     @include f.font-size(common.$nav-text-size);
     color: common.$nav-text-color;
     font-weight: f.$font-weight-regular;
+
     @include f.media('>mid') {
         font-weight: common.$nav-text-weight;
     }
+    @include f.media('>mid', '<wide') {
+        max-width: 200px;
```

Comment:
I think you can use one of our [mixins](https://github.com/justeat/fozzie/blob/514a759cf98081895fdfa7556742b0a07248de43/src/scss/tools/mixins/_truncate.scss#L8) here?

Reply:
I didn't realise we had that 😆  I'll switch that out and put a PR in for it